### DISCUSSION TITLE
How to specify the stride in setting

### DISCUSSION THREAD

**Discussion by chengzhanhong** [PaddlePaddle/PaddleSpatial#194] [BaiduKDDCup2022]
The dataset report says "A length-𝐿𝑥-length-288 prediction window is adopted to roll the whole test set with stride Δ𝑡 time step". 

However, there is no code about the `stride` in [evaluation.py](https://github.com/PaddlePaddle/PaddleSpatial/blob/b8bdee9698b7f7568cb9508586932947b076ea05/apps/wpf_baseline_gru/kddcup22-sdwpf-evaluation/paddlepaddle/evaluation.py).

I am wondering if tests with different strides are just different test files, as in the current implementation: https://github.com/PaddlePaddle/PaddleSpatial/blob/b8bdee9698b7f7568cb9508586932947b076ea05/apps/wpf_baseline_gru/kddcup22-sdwpf-evaluation/paddlepaddle/evaluation.py#L167

which means the submitted forecast function does not need to consider the `stride` parameter?

**Comment by ramber1836** [comment]
No, you don't need to consider the stride parameter.

**Comment by zhoujingbo** [comment]
Hi,

Yes, the stride is set by the organizers. 

Thanks a lot for your support.
Regards
